core/citrus-spring/src/main/java/com/consol/citrus/common/XmlTestLoader.java

```diff
@@ -98,6 +111,26 @@ private ApplicationContext loadApplicationContext() {
         }
     }
 
+    /**
+     * Configures the CitrusNamespaceParserRegistry with custom parsers
+     */
+    private void configureCustomParsers() {
+        XmlTestLoaderConfiguration loaderConfiguration = testClass.getAnnotation(XmlTestLoaderConfiguration.class);
+        if (loaderConfiguration != null) {
+            Arrays.stream(loaderConfiguration.parserConfigurations()).forEach(testCaseParserConfiguration -> {
```

Comment:
I think streams is superfluous at this point? just use `for(TestCaseParserConfiguration testCaseParserConfiguration: loaderConfiguration.parserConfigurations())`..

Reply:
As old school programmer it took me a while to adapt to streams but now I am streaming everything ;-).

Reply:
it's not always "the yellow of the egg" 😉